### PR TITLE
extension with MagicPython scopes

### PR DESCRIPTION
**Pull request by bfelder**
This pull request is a basis for discussion. Everything may still be changed.

- diffs only affect python, other syntaxes are not touched at all.
- added those scopes that were newly introduced from MagicPython.
**- scheme works with both standard Python and MagicPython**
- the python color scheme of the original Seti UX  has remained largely the same
- I grouped scopes as I saw fit. For example grouping start and end scopes, and scopes for punctuation.
- whenever possible I tried make colors the same in MagicPython as they have been in standard Python
- yet it is important to note that MagicPython redefines certain scopes, so they capture different keywords, therefore identical coloring is not always possible
- also I have made use of new scopes to implement a finer more distinguished resolution of the python syntax. For example now the important self keyword has its own scope. As the main advantage of Seti UX is enhanced comprehensibility of code via pronounced coloring. We should build on this strength here and give those scopes their own colors. Really making python language come alive.
 **List of some important new scopes:**
> constant.numeric.float.python 
> punctuation.definition.list.begin.python, punctuation.definition.list.end.python
> keyword.operator.comparison.python 
> variable.language.special.cls.python 
> variable.language.special.self.python,
> variable.parameter.function.language.special.self.python

- Some special notes:
1. **support.type.exception.python** should have a color stressing the role of an exception
2. **variable.legacy.builtin.python** for deprecated keywords
3. **meta.function-call.python** for *args, **kwargs, should have own color to contrast with positional arguments
4. **punctuation.definition.list**, **punctuation.definition.dict** for [], {} such central data structure in python should made recognizable with own color, not just white (one of few suggestions regarding default scheme)
5. comparison operators can now have own color, could help to reduce bugs if it differed from assignments

**ToDo**
- colors need to be adjusted, the sopes were given random colors which are neither beautiful nor promote legibility
_- test files covering virtually all aspects of python syntax can be found in the MagicPython repository itself: [https://github.com/MagicStack/MagicPython/tree/master/test](MagicPython)_

### CODE REVIEW

**Comment by ctf0** [comment]
good job, however should i merge it or wait till you finish adjusting the colors ?

**Comment by bfelder** [comment]
I did express myself not clearly, I am afraid:
**I would actually like you to chose appropriate colors.**

You are the creator of this beautiful theme and have thereby proven your aptitude to select beautiful and highly legible color palettes. Starting from the randomly generated colors I have tried to refine those, but only ended up with colored syntax not as pretty as the original.

You can edit the pull request. I give you my opinion on your color choices.
Then you can merge.

**Comment by ctf0** [comment]
thanx for the kind words, the prob is i have no idea how to use python and what colors should represent with each scope, so to solve this

1- install https://packagecontrol.io/packages/ColorHelper which would show you a preview of each color in the scheme file
2- choose the colors that looks appropriate to you and apply it to the new syntax

**Comment by bfelder** [comment]
Thanks. That is a lot of trust you put in me. I am going to determine the look and feel of one Sublime's most popular color schemes after all and thousands of users will be influenced by this.

I am going need a week to make sure everything looks as good as possible.

You will hear from me on next Sunday.

File: Seti.tmTheme [hunk 1]
```diff
@@ -23,7 +23,7 @@
                     <key>invisibles</key>
                     <string>#F3FFB51A</string>
                     <key>lineHighlight</key>
-                    <string>#7292a130</string>
```
Comment (bfelder) [inline Seti.tmTheme]: Slightly brightened line highlight to make it more recognizable.

**Comment by bfelder** [comment]
@ctf0 Hey Muah I worked the whole day on the scheme. It is now good. It created the one I want to do python 3 coding with, still stick to the original color choices you made as possible.

File: Seti.tmTheme (hunk 1 above)
Comment (ctf0) [inline Seti.tmTheme]: i prefer if you keep the LH as it is and you can modify ur local copy later

Comment (bfelder) [inline Seti.tmTheme]: Sure. Just a suggestion. Skip this diff.

File: Seti.tmTheme [hunk 2]
```diff
@@ -444,7 +444,7 @@
                 <key>settings</key>
                 <dict>
                     <key>foreground</key>
-                    <string>#009FFF</string>
```
Comment (bfelder) [inline Seti.tmTheme]: Skip this as well.

Comment (ctf0) [inline Seti.tmTheme]: i think you need to edit it from ur side and update the PR ?

Comment (bfelder) [inline Seti.tmTheme]: Give me a moment. I'll do the changes locally then push again.

Comment (bfelder) [inline Seti.tmTheme]: Or better keep it. This line belongs to the scope: **keyword.control.import.python** I feel the import section needs a distinct color as well. Also good to note stray import statements throughout the code, like those in pdb breakpoint (import pdb, pdb.set_trace())

**Comment by bfelder** [comment]
Cool. :-)

**Pull request merged**